suites/quincy/rbd/tier-3_rbd_persistent_write_back_cache.yaml

```diff
@@ -461,3 +461,27 @@ tests:
       module: test_performance_pwl_cache.py
       name: Performance comparison with or without pwl cache
       polarion-id: CEPH-83574711
+
+  - test:
+      abort-on-fail: true
+      config:
+        level: client                        # PWL at client
```

Comment:
Can we test this at client, pool and image levels as well. Refer previous tests for code details for the same.

Reply:
@manasagowri Since there are cluster operations in which the client is performing, when the tests are repeating over paralllely  with all levels it fails, which are an intermittent issue, also this cluster-related operation doesn't affect pwl settings at `pool`, `image` other than majorly on `client` level,
hence i added test only to the client level
Test result: failed intermittent issue at all levels
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-8HSZXV

Reply:
The issue mentioned in the above link looks more like an automation issue for me. Could you please debug further?

Reply:
As discussed please add retries and verify this.

Reply:
Covered test case with client, pool and image level.
thanks, @manasagowri for the help in debugging Initially i was using conf for testing was wrong also added retries for it.
success logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-TP485L/